src/workerd/server/workerd-api.c++

```diff
@@ -234,6 +235,7 @@ WorkerdApi::EntrypointClasses WorkerdApi::getEntrypointClasses(jsg::Lock& lock)
   return {
     .workerEntrypoint = typedLock.getConstructor<api::WorkerEntrypoint>(lock.v8Context()),
     .durableObject = typedLock.getConstructor<api::DurableObjectBase>(lock.v8Context()),
+    .workflow = typedLock.getConstructor<api::Workflow>(lock.v8Context()),
```

Comment:
This will need to land together with an internal PR making the same change to the internal version of this file, otherwise I think the edge runtime will crash due to this being left null.